README.md

```diff
@@ -200,10 +200,16 @@ The access token is typically set in the `nx.json` file in your project. You can
 
 ```
 "nxCloud": {
-  "accessToken": "<access-token>"
+  "nxCloudAccessToken": "<access-token>"
```

Comment:
why did you change the name field?

Reply:
It was different in: https://nx.dev/ci/recipes/security/access-tokens